tests/_data/AntiMalwareTestFile.txt

```diff
@@ -0,0 +1 @@
+X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*
```

Comment:
@jack7anderson7 Thinking about this further, we should probably remove this. We don't publish tests with release packages but this is still going to impact anyone pulling down from the repo directly (Setting off AV, etc).

If we need to test this we'll likely need to look at mocking instead.